PennMobile/Auth/PennAuthRequestable.swift

```diff
@@ -43,7 +43,8 @@ extension PennAuthRequestable {
                 }
             } else {
                 UserDefaults.standard.setShibbolethAuth(authedIn: false)
-                completionHandler(nil, nil, NetworkingError.authenticationError)
+//                completionHandler(nil, nil, NetworkingError.authenticationError)
+                completionHandler(nil, nil, NetworkingError.noInternet)
```

Comment:
This isn't an accurate way of checking for no internet. Let's use [this example](https://github.com/pennlabs/penn-mobile-ios/blob/master/PennMobile/Home/Networking/HomeAPIService.swift#L40) to check for no internet at the very beginning of the datatask callback. The issue with failing with no internet here is that it may fail for a variety of reasons, such as the server not returning a valid HTTP response or the data not decodable as a string. Instead of passing back a `noInternet` error though, let's pass back an `other` error here.